docs/references/django_model_types.md

```diff
@@ -0,0 +1,49 @@
+# Django Model Types
+
+Django Models can be converted in Strawberry Types with the `@strawberry_django.type` decorator
+
+```python
+import strawberry
+
+@strawberry.django.type(models.Fruit)
+class Fruit:
+    ...
+```
+
+## Fields
+
+By default, no fields are implemented on the new type. For details on adding fields,
+see the [Fields](fields.md) documentation.
+
+## Queryset setup
+
+By default, a Strawberry Django type will pull data from the default manager for it's Django Model.
+You can change this behavior by implementing a `get_queryset` method on the type.
```

Comment:
Don't know if "change this behavior" is the correct phrase here. The reason one would override this would be to do some extra filtering, do some annotation or something like that.

Maybe rewrite this to something like "You can implement a `get_queryset` to your type to do some extra processing to the queryset, like further filtering it."